### PR TITLE
Retrieve probabilities from detections.

### PR DESCRIPTION
**Pull request by ooobelix**

### CODE REVIEW

File: arapaho/test.cpp [hunk 1]
```diff
@@ -224,8 +232,10 @@ int main()
                     // Show labels
                     if (labels[objId].c_str())
                     {
-                        DPRINTF("Label:%s\n\n", labels[objId].c_str());
-                        putText(image, labels[objId].c_str(), cvPoint(leftTopX, leftTopY),
+                        char str[200];
+                        sprintf(str, "%s: %f", labels[objId].c_str(), probs[objId]);
```
Comment (prabindh) [inline arapaho/test.cpp]: Can we use snprintf (secure variants) instead please ?

File: arapaho/test.cpp [hunk 2]
```diff
@@ -224,8 +232,10 @@ int main()
...
+                        DPRINTF("Label:%s, Probabilty: %f\n\n", labels[objId].c_str(), probAsString);
```
Comment (prabindh) [inline arapaho/test.cpp]: Where is probAsString declared ?

**Review by prabindh — CHANGES_REQUESTED** [review]
@ooobelix  Could you please respond to the comments ? Thanks

File: arapaho/test.cpp (hunk 2 above)
Comment (ooobelix) [reply arapaho/test.cpp]: you are right, sorry! It's "char str[200]; ", fixed in the same branch!

File: arapaho/test.cpp (hunk 1 above)
Comment (ooobelix) [reply arapaho/test.cpp]: Fixed, thanks for the review

**Review by prabindh — APPROVED** [review]

**Pull request merged**